src/skottie-player.ts

```diff
@@ -71,6 +76,27 @@ export class SkottiePlayer extends LitElement {
         max-width: 100%;
         font-family: sans-serif;
       }
+
+      .layerbutton {
+        background-color: #eee;
+        color: #444;
```

Comment:
Is there a style guide that we should put all 6 digits for the color value, or 3 digits would be fine too? I'm referring to #444.

Reply:
According to the css style guide shorthand are the recommended style